src/api/get-installations.js

```diff
@@ -41,7 +41,10 @@ async function findWindowsBrowsers () {
     var browserRe     = new RegExp(regKeyEsc + '([^\\\\]+)\\\\shell\\\\open\\\\command' +
                                    '\\s+\\([^)]+\\)\\s+reg_sz\\s+([^\n]+)\n', 'gi');
 
-    var stdout = await exec(`chcp 65001 | reg query ${regKey} /s`);
+    var stdout       = await exec('chcp');
+    var origCodePage = stdout.match(/\d{1,5}/)[0];
+
+    stdout = await exec(`chcp 65001 | reg query ${regKey} /s`);
```

Comment:
```js
// NOTE: we should change code page to get correct result regardless the WIndows localization.
// Then we should restore it to avoid the terminal errors.
```
Add the comment here and it's ok, create a pull request to the upstream repo.